### ISSUE TITLE
Allow Impact to run without Baritone

### ISSUE THREAD

**Issue by ZeroMemes** [ImpactDevelopment/ImpactClient#1544]
Baritone is a fantastic addon to Impact, and one that is heavily utilized, however I believe that Impact should not be dependent on it. I suggest making a separate `baritone` sourceset, which includes Baritone as a dependency, implements the various features that Impact provides as a way to interface with Baritone, but is not required for Impact itself to function.

**Comment by LeafHacker** [comment]
Agreed. Alternatively we could make baritone a `provided` dependancy and just check for its existence at runtime. That way to install baritone you simply add the API library to your Impact json

**Comment by VirxEC** [comment]
In the installer, add an option for the baritone version, just like with optifine. That way we can always have the latest version.
